optimum/intel/openvino/trainer.py

```diff
@@ -153,6 +153,7 @@
         "{re}.*Embedding.*",
         "{re}.*add___.*",
         "{re}.*layer_norm_.*",
+        "{re}.*scaled_dot_product_attention_0",
```

Comment:
In https://github.com/openvinotoolkit/nncf/pull/2761, support for SDPA was added. However, it does not work properly for `create_compressed_model` API. Adding to ignored scope for now. This does not change the resulting number of FQ nodes, because SDPA was not quantized before too. Ticket: 148687.

cc @AlexanderDokuchaev